frame/composable-support/src/types/mod.rs

```diff
@@ -60,3 +89,13 @@ impl sp_std::fmt::Debug for EcdsaSignature {
 		write!(f, "EcdsaSignature({:?})", &self.0[..])
 	}
 }
+
+impl From<CosmosEcdsaSignature> for EcdsaSignature {
+	fn from(item: CosmosEcdsaSignature) -> Self {
+		let mut sig = item.0.to_vec();
```

Comment:
Can we remove this allocation here?

Reply:
24f6d1cdc04da7916fff83a71b7a86dcba1408fb